src/commands/go-test.yaml

```diff
@@ -45,11 +45,11 @@ steps:
       name: Check if dependencies have known security vulnerabilities
       command: |
         set +e
-        CGO_ENABLED=0 go list -json -m all | nancy sleuth --skip-update-check --quiet --exclude-vulnerability-file ./.nancy-ignore --additional-exclude-vulnerability-files ./.nancy-ignore.generated | tee ./nancy-results.txt ; nancy_result=(${PIPESTATUS[1]})
+        CGO_ENABLED=0 go list -json -m all | nancy sleuth --skip-update-check --quiet --exclude-vulnerability-file ./.nancy-ignore --additional-exclude-vulnerability-files ./.nancy-ignore.generated 2>&1 | tee ./nancy-results.txt ; nancy_result=(${PIPESTATUS[1]})
         grep -q 'error accessing OSS Index' nancy-results.txt; grep_result=$?
         set -e
         # If nancy gave us a bad exit code AND grep found an OSS index error in the output, then we don't fail the build.
-        if [[ $nancy_result -eq 1 && $grep_result -eq 0 ]]; then
```

Comment:
Now that I think more about it, I vaguely remember the exit code was significant. Perhaps nancy uses another exit code when it finds vulnerabilities? Did you check that this won't ignore failures due to normal scans with vulnerabilities?

Reply:
It's `&& $grep_result -eq 0`, normal nancy reported issues wouldn't have `error accessing OSS Index` in the output, right!?

Reply:
Like it is now, it would even work if a new `nancy` version outputs its `error accessing OSS Index` and then segfaults (exit code 139). 😂

Reply:
You're right, this is probably fine. My wires are still crossed from trying to get this into a `sed` one liner just to find out busybox doesn't support the `sed` behavior I needed :dizzy_face: